node/rnode.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=RChain Node
+After=network.target
+
+[Service]
```

Comment:
Looks pretty good. People rip on systemd but I love the defaults and ti takes care of the rest. What about running as user rnode? I'm assuming we want this right?
[Service]
User=rnode

Reply:
Yes definitely.